src/player.py

```diff
@@ -1,2 +1,30 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+
+
+class Player:
+    def __init__(self, name, loc):
+        self.name = name
+        self.loc = loc
+
+    def act(self, dir):
+        target = getattr(self.loc, dir)
+        if target:
+            self.loc = target
+            return f'You moved into {self.loc.name}...'
+        else:
+            return False
+
+    def situation(self):
```

Comment:
I'm assuming that when you call player.situation(), you get a list of all possible moves and what rooms they lead to?
which would make sense with your Room class having n_to (and the rest) set to None